daemon/algod/api/algod.oas2.json

```diff
@@ -1235,6 +1235,65 @@
         }
       }
     },
+    "/v2/transactions/async": {
+      "post": {
+        "tags": [
+          "experimental"
```

Comment:
:+1: 

This tag makes it so the endpoint wont show up in the SDKs:
https://github.com/algorand/generator/blob/master/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java#L471